aten/src/ATen/native/native_functions.yaml

```diff
@@ -6496,6 +6496,21 @@
   dispatch:
     DefaultBackend: hypot_
 
+- func: igamma.out(Tensor self, Tensor other, *, Tensor(a!) out) -> Tensor(a!)
+  dispatch:
+    CPU, CUDA: igamma_out
+
+- func: igamma(Tensor self, Tensor other) -> Tensor
+  use_c10_dispatcher: full
+  variants: method, function
+  dispatch:
+    DefaultBackend: igamma
```

Comment:
I think this should be CPU, CUDA: here and below, following the updated note. Otherwise this will attempt to dispatch for XLA, which is probably why those odd XLA errors were appearing (an error would still appear, but it should be clearer if XLA doesn't have the function implemented).

Reply:
Done